example/centos/centos-9-ppc64le-tar.yaml

```diff
@@ -9,7 +10,7 @@ otk.define:
   packages:
     build:
       otk.external.osbuild-gen-depsolve-dnf4:
-        architecture: s390x
+        architecture: ${architecture}
         module_platform_id: c9s
         releasever: "9"
         repositories:
```

Comment:
repositories below contians still a hardcoded "ppc64le" value (but only here and not in the other arches it seems)?

Reply:
Fixed. Probably misgrepped :)